src/Aspire.Hosting/DistributedApplication.cs

```diff
@@ -52,6 +146,38 @@ public static IDistributedApplicationBuilder CreateBuilder(string[] args)
     /// </summary>
     /// <param name="options">The <see cref="DistributedApplicationOptions"/> to use for configuring the builder.</param>
     /// <returns>A new instance of the <see cref="IDistributedApplicationBuilder"/> interface.</returns>
+    /// <remarks>
+    /// <para>
+    /// The <see cref="DistributedApplication.CreateBuilder(DistributedApplicationOptions)"/> method provides
+    /// greater control over the behavior of the distributed application at runtime. For example using providing
+    /// a <paramref name="options"/> argument allows developers to force all container images to be loaded
+    /// from a specified container registry by using the <see cref="DistributedApplicationOptions.ContainerRegistryOverride"/>
+    /// property, or disable the dashboard by using the <see cref="DistributedApplicationOptions.DisableDashboard"/>
+    /// property. Refer to the <see cref="DistributedApplicationOptions"/> class for more details on
+    /// each option that may be provided.
+    /// </para>
+    /// <para>
+    /// When supplying a custom <see cref="DistributedApplicationOptions"/> it is commended to populate the
+    /// <see cref="DistributedApplicationOptions.Args"/> property to ensure that the AppHost continues to function
+    /// correctly when used with deployment tools that need to enable publish mode.
+    /// </para>
+    /// </remarks>
+    /// <example>
+    /// Override the container registry used by the distributed application.
+    /// <code>
+    /// var options = new DistributedApplicationOptions
```

Comment:
Noticed that the spacing is 2, I would prefer 4 for readability.

Reply:
Done.